README.md

```diff
@@ -238,17 +238,14 @@ python setup.py install    # install cugraph python bindings
     # Run the cugraph tests
     cd $CUGRAPH_HOME
     cd cpp/build
-    make test
-    
-    # alternatively, you can run individual test
     gtests/GDFGRAPH_TEST		# this is an executable file

```

Comment:
If we're running just gtests/GDFGRAPH_TEST, shouldn't we better remove other tests? (Sorry for bringing this issue on this pull request's review... This may be a separate issue, but just in case if you can quickly fix this). cpp/src/tests/nvgraph_plugin/nvgraph_gdf_sssp.cpp and cpp/src/test/pagerank/pagerank_test.cu still have absolute paths to the test files.
e.g.
pagerank_test.cu
```cpp
188    ::testing::Values(  Pagerank_Usecase("/datasets/networks/karate.mtx", "")
```
gtests/PAGERANK_TEST leads to Segmentation fault. This has been like this for quite a while, and if we are not planning to fix this, I think it's better to just remove these tests.

Reply:
This is a separate issue. 
`make test` should not be documented until we can provide access to the C++ datasets.

Reply:
I think it still worth running `gtests/GDFGRAPH_TEST` as this verifies that libcugraph.so is not corrupted (independently of any python things).